### DISCUSSION TITLE
Have a restart option while already capturing

### DISCUSSION THREAD

**Discussion by avimaymon** [NickeManarin/ScreenToGif#1160] [Ideas]
Many times, I've found myself stopping a capture, waiting for processing only to then start a new capture and discard the old changes, would've been really nice to have a restart option while capturing

**Comment by NickeManarin** [comment]
There's this option:

![image](https://user-images.githubusercontent.com/14798947/205494350-40a5acd1-fd53-4565-a21c-28c2834b7554.png)

In use: 

![image](https://user-images.githubusercontent.com/14798947/205494383-2b800495-c908-4abe-8997-e81e4094b499.png)

You can just hit discard while recording.
Or you can just pause the capture and hit discard normally.
It will "reset" the recording.

**Reply by avimaymon** [reply]
NICEEEE, thanks!
